cost-analyzer/templates/cost-analyzer-network-costs-service-template.yaml

```diff
@@ -9,8 +9,8 @@ metadata:
     prometheus.io/scrape: "{{ .Values.networkCosts.prometheusScrape }}"
     prometheus.io/port: {{ (quote .Values.networkCosts.port) | default (quote 3001) }}
   labels:
-    app: {{ template "cost-analyzer.networkCostsName" . -}}
     {{ include "cost-analyzer.commonLabels" . | nindent 4 }}
+    app: {{ template "cost-analyzer.networkCostsName" . -}}
```

Comment:
ending `-` breaks yaml. one will get 
```
Error: YAML parse error on cost-analyzer/templates/cost-analyzer-network-costs-service-template.yaml: error converting YAML to JSON: yaml: line 15: mapping values are not allowed in this context
helm.go:81: [debug] error converting YAML to JSON: yaml: line 15: mapping values are not allowed in this context
YAML parse error on cost-analyzer/templates/cost-analyzer-network-costs-service-template.yaml
```

Reply:
Thanks I removed the '-' char in latest commit and tested deploying on a cluster.